BatchCave.py

```diff
@@ -2114,41 +2119,8 @@ def ER_WestlawNext(self, x, name='ER-WestlawNext'):
         x = utilities.MarcEditMakeFile(x)
         return x
 
-    def ER_WoltersKluwer_Cheetah(self, x, name='ER-WoltersKluwer-Cheetah'):
-        print '\nRunning change script '+ name + '\n'
-        x = utilities.MarcEditBreakFile(filename)
-        #Change =001 field to =002, and add =003
-        x = re.sub('(?m)^=001  (.*)', '=002  cheetah_\\1\n=003  ER-WlK-Cheetah', x)
-        #ADD 730, 949 fields before supplied 008
-        x = re.sub('(?m)^=008', r'=949  \\1$luint$rs$t99\n=949  \\\\$a*b3=z;bn=buint;\n=730  0\\$aWolters Kluwer.$pCheetah.$5OCU\n=506  \\\\$aLicensed for use only by University of Cincinnati law faculty and students.\n=008', x)
-        #Add customized $3 and standard $z at end of 856
-        x = re.sub('(=856.*)', '\\1$3?Cheetah: Access limited to UC Law faculty and students : $zConnect to resource', x)
-        x = utilities.DeleteLocGov(x)
-        x = utilities.AddEresourceGMD(x)
-        x = utilities.CharRefTrans(x)
-        x = utilities.MarcEditSaveToMRK(x)
-        x = utilities.MarcEditMakeFile(x)
-        return x
-
-    def ER_WoltersKluwer_IntelliConnect(self, x, name='ER-WoltersKluwer-IntelliConnect'):

```

Comment:
Also this one?

Reply:
Actually I'm not sure why these are all missing. I started at the top of the list of functions. These are the ones I broke out into modules. I actually did put them back aka start from the original script. The only thing I can think are these are edits Jeff made and sent out before I could use Github. I'll go back to my emails and see what changes were made.